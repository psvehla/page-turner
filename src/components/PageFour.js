import React from 'react';
import logo from '../images/logo.svg';

import './PageFour.css';

function PageFour(props) {
  return (
    <div><img src={logo} alt="logo" onClick={() => props.onClick()} /></div>
  );
}

export default PageFour;
