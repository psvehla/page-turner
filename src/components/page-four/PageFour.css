* {
  margin: 0;
  padding: 0;
}
.imgbox {
  display: grid;
  height: 100%;
}
.centre-fit {
  max-width: 100%;
  max-height: 100vh;
  margin: auto;
}
