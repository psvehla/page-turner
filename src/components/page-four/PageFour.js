import React from 'react';
import bike from '../../images/4.jpg';

import './PageFour.css';

function PageFour(props) {
  return (
    <div className="imgbox">
      <img className="centre-fit" src={bike} alt="bike" onClick={() => props.onClick()} />
    </div>
  );
}

export default PageFour;
