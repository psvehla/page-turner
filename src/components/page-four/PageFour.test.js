import React from 'react';
import renderer from 'react-test-renderer';
import PageFour from './PageFour';

it('renders correctly', () => {

  const tree = renderer
    .create(<PageFour />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
