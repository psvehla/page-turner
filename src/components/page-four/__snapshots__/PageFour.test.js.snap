// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="imgbox"
>
  <img
    alt="bike"
    className="centre-fit"
    onClick={[Function]}
    src="4.jpg"
  />
</div>
`;
